markdown/blog/2024-gsoc-wrap.md

```diff
@@ -0,0 +1,112 @@
+title: "2024 Google Summer Of Code Wrap Up"
+date: 2024-12-11T06:00:00+01:00
+type: Community
+tags:
+  - Mentorship
+  - GSoC
+  - AsyncAPI
+  - OSS
+cover: /img/posts/gsoc.webp
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: AsyncAPI Preacher 
+---
+
+For the first time in our history with Google Summer of Code (GSoC), AsyncAPI proudly participated as an independent organization. In previous years, we had joined the program under Postman’s umbrella due to the challenges of being selected as a participating organization. With Postman’s higher chances of acceptance, they graciously allowed us to include some of our projects under their organization. This year, however, marked a significant milestone as AsyncAPI stepped into the spotlight on its own. 
+
+The goal of this post is to share our GSoC journey and offer valuable insights for future organizations and contributors. By reflecting on what went well, the challenges we encountered, and the lessons learned, we hope to help others make the most of this incredible program.
+
+Now, let’s dive into the key achievements and challenges of our participation this year.
+
+## Key Achievements
+
+Without a doubt, our biggest achievement this year has been participating in GSoC as an individual organization. For several years, AsyncAPI applied as a standalone organization but was never selected.
+
+So, what changed this time? We took a step back to analyse potential reasons for our previous rejections and used those insights to refine our approach. One significant change was starting our preparation well ahead of the deadline. Special thanks go to [Azeez Elegbede](https://github.com/AceTheCreator), who led the effort by curating project ideas, crafting our organization application, reaching out to potential mentors, and engaging with interested contributors. Azeez worked closely with the incredible [Quetzalli](https://github.com/quetzalliwrites), whose expertise played a significant role in reviewing the application and project ideas, providing valuable guidance and support throughout the process.
+
+To showcase the impact of this year’s participation, let’s look at some key community accomplishments achieved during GSoC 2024.
+
+- This year, we achieved our highest number of project idea slots in the history of our GSoC participation, with a total of **7 slots**.
+- For the first time, we had **4 ex-GSoC participants** return to the program as mentors, marking a significant milestone in building a cycle of mentorship and growth within our community.
+- We received an impressive **77 project proposals**, the most we've ever had in our GSoC journey.
+- Over **100 new contributors** joined our community from GSoC, showcasing the growing interest and engagement with AsyncAPI.
+- AsyncAPI proudly participated as an organization at the **GSoC Summit in San Francisco**, further showcasing our presence in the open-source ecosystem.
```

Comment:
you can mention that we did not participate only as listeners but I actually delivered a lightning talk about maintainership-proposal concepts